roles/convert/tasks/main.yml

```diff
@@ -23,4 +27,9 @@
     reboot_timeout: 120
   timeout: 180
   when: convert_reboot_requested
+
+- name: Remove temporary configuration file
+  ansible.builtin.file:
+    name: "{{ convert2rhel_configfile.path }}"
+    state: absent
 ...
```

Comment:
As @andywaltlova mentioned in her review, I think we can move this to the common role later and make it a handler, just to remove this duplicate instruction from convert/analysis. 

But for now is all good.